pkg/webui/account/containers/clients-table/index.js

```diff
@@ -151,7 +151,7 @@ const ClientsTable = () => {
         }),
         render: details => (
           <ButtonGroup align="end">
-            <Button message={sharedMessages.restore} onClick={details.restore} />
+            <Button message={sharedMessages.restore} onClick={details.restore} secondary />
```

Comment:
This change has to do with a change of how the button works. I believe it's best to always explicitly pass the button style (`primary`, `secondary`, `naked`) and apply no styling at all when no style prop is set, instead of relying on `unstyled` or `raw` props.